### PR TITLE
[BZ1273470] Camel quickstart missing javax.transaction.TransactionMan…

### PR DESCRIPTION
**Pull request by Holmistr**
…ager

See: https://bugzilla.redhat.com/show_bug.cgi?id=1273470

### CODE REVIEW

**Comment by Holmistr** [comment]
I'm sorry, there is a typo in the commit message and PR name. It's in the carmart quickstart, not camel.

**Comment by mgencur** [comment]
Yeah, it's the same type of typo like when you go to a restaurant, want to order coke, then you make four typos and order a beer:)

**Comment by Holmistr** [comment]
@mgencur I'm glad that I'm not the only one who experienced such troubles :D

File: carmart/pom.xml [hunk 1]
```diff
@@ -336,6 +336,13 @@
                     <scope>compile</scope>
                 </dependency>
 
+                <!-- Import the JTA API -->
+                <dependency>
+                    <groupId>org.jboss.spec.javax.transaction</groupId>
+                    <artifactId>jboss-transaction-api_1.1_spec</artifactId>
+                    <scope>compile</scope>
```
Comment (slaskawi) [inline carmart/pom.xml]: Can we remove the scope? Compiled is the default...

File: carmart/pom.xml [hunk 2]
```diff
@@ -433,6 +440,13 @@
...
```
Comment (slaskawi) [inline carmart/pom.xml]: Same here

**Comment by slaskawi** [comment]
Looks good, integrating

**Comment by slaskawi** [comment]
Integrated thanks!

**Pull request merged**